CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 2.2.1 (Next)
 
 * Your contribution here.
+* [#180](https://github.com/slack-ruby/slack-ruby-bot-server/pull/180): grape v3 compatibility - [@markokajzer](https://github.com/markokajzer).
```

Comment:
This will fail Danger (danger is being fixed in https://github.com/ruby-grape/grape/issues/2630).


```suggestion
* [#180](https://github.com/slack-ruby/slack-ruby-bot-server/pull/180): Add support for Grape v3 - [@markokajzer](https://github.com/markokajzer).
```